### ISSUE TITLE
grails wrapper cannot work on cygwin with sdkman.

### ISSUE THREAD

**Issue by abcfy2** [grails/grails-core#10869]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [ ] Example that reproduces the problem uploaded to Github
- [x] Full description of the issue provided (see below)

### Steps to Reproduce
1. use cygwin and install sdkman `curl -s "https://get.sdkman.io" | bash`
2. installed gradle by using sdkman: `sdk i gradle`
3. use grails wrapper to build project: `grails create-app test && cd test && ./grailsw war`
4. will see this error: `Error: Unable to access jarfile /cygdrive/c/Users/abcfy/projects/test/grails-wrapper.jar
`, but I've already fixed https://github.com/grails/grails-wrapper/pull/3 (not merged yet.)
5. After fix cygwin path issue: `./grailsw war`, will see this error:
```
|Resolving Dependencies. Please wait...
Error |
Error initializing classpath: The specified Gradle installation directory 'C:\Users\abcfy\.sdkman\candidates\gradle\current' is not a directory. (NOTE: Stack trace has been filtered. Use --verbose to see entire trace.)
java.lang.IllegalArgumentException: The specified Gradle installation directory 'C:\Users\abcfy\.sdkman\candidates\gradle\current' is not a directory.
        at org.gradle.tooling.internal.consumer.DistributionFactory$InstalledDistribution.getToolingImpl(DistributionFactory.java:239)
        at org.gradle.tooling.internal.consumer.DistributionFactory$InstalledDistribution.getToolingImplementationClasspath(DistributionFactory.java:228)
        at org.gradle.tooling.internal.consumer.loader.CachingToolingImplementationLoader.create(CachingToolingImplementationLoader.java:39)
        at org.gradle.tooling.internal.consumer.loader.SynchronizedToolingImplementationLoader.create(SynchronizedToolingImplementationLoader.java:42)
        at org.gradle.tooling.internal.consumer.connection.LazyConsumerActionExecutor.onStartAction(LazyConsumerActionExecutor.java:95)
        at org.gradle.tooling.internal.consumer.connection.LazyConsumerActionExecutor.run(LazyConsumerActionExecutor.java:78)
        at org.gradle.tooling.internal.consumer.connection.CancellableConsumerActionExecutor.run(CancellableConsumerActionExecutor.java:45)
        at org.gradle.tooling.internal.consumer.connection.ProgressLoggingConsumerActionExecutor.run(ProgressLoggingConsumerActionExecutor.java:58)
        at org.gradle.tooling.internal.consumer.connection.RethrowingErrorsConsumerActionExecutor.run(RethrowingErrorsConsumerActionExecutor.java:38)
        at org.gradle.tooling.internal.consumer.async.DefaultAsyncConsumerActionExecutor$1$1.run(DefaultAsyncConsumerActionExecutor.java:55)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)

Error |
Error initializing classpath: The specified Gradle installation directory 'C:\Users\abcfy\.sdkman\candidates\gradle\current' is not a directory.
```

### Environment Information

- **Operating System**: Windows 10 with Cygwin
- **Grails Version:** 3.3.1
- **JDK Version:** 1.8
- **Container Version (If Applicable):** TODO

**Comment by abcfy2** [comment]
Maybe same issue with https://github.com/grails/grails-core/issues/10207, but I think it's not fixed yet. And hardcodes is not a good idea. sdkman will set `GRADLE_HOME` environment variable, maybe it's better to use `GRADLE_HOME` instead use hardcodes `current` directory?

I will try to make a pull request to fix this issue.

**Comment by ilopmar** [comment]
This was fixed and released in Grails 3.3.3 by https://github.com/grails/grails-core/pull/10908
